addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -606,6 +608,17 @@ export function makeDraggableHook(hookParams) {
                 }
             };
 
+            /**
+             * Global (= ref) "click" event handler.
+             * Used to prevent click events after dragEnd
+             * @param {PointerEvent} ev
+             */
+            const onClick = (ev) => {
+                if (preventClick) {
+                    safePrevent(ev, { stop: true });
```

Comment:
no test? A concrete example in the commit message would be nice

Reply:
not really a fix. In fact the feature was initially developed for moving elements. Here we use it to select cells and we don't have any element, just a cursor